apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1.Tests/ResourcePrefixHeaderTest.cs

```diff
@@ -0,0 +1,184 @@
+// Copyright 2020, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Grpc.Core;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace Google.Cloud.Spanner.V1.Tests
+{
+    public class ResourcePrefixHeaderTest
+    {
+        const string header = "google-cloud-resource-prefix";
+        const string database = "projects/proj/instances/inst/databases/db";
+        const string session = "projects/proj/instances/inst/databases/db/sessions/sess";
```

Comment:
```suggestion
        private const string ResourcePrefixHeader = "google-cloud-resource-prefix";
        private const string SampleDatabaseName = "projects/proj/instances/inst/databases/db";
        private const string SampleSessionName = "projects/proj/instances/inst/databases/db/sessions/sess";
```

(If you're using Visual Studio, use F2 to do the renaming and all the uses will change too.)